core/main/src/firebolt/rpc_router.rs

```diff
@@ -134,15 +134,23 @@ async fn resolve_route(
 impl RpcRouter {
     pub async fn route(
         state: PlatformState,
-        req: RpcRequest,
+        mut req: RpcRequest,
         session: Session,
         timer: Option<Timer>,
     ) {
         let methods = state.router_state.get_methods();
         let resources = state.router_state.resources.clone();
 
+        let method = req.method.clone();
+        if let Some(overriden_method) = state
+            .get_manifest()
+            .has_rpc_override_method(&req.method)
+        {
+            req.method = overriden_method;
+            println!("route: overridding ")
```

Comment:
Remove the println we can probably add a proper debug here

Reply:
Removed